crates/debugger_ui/src/session/running/stack_frame_list.rs

```diff
@@ -250,6 +250,9 @@ impl StackFrameList {
         let Some(abs_path) = Self::abs_path_from_stack_frame(&stack_frame) else {
             return Task::ready(Err(anyhow!("Project path not found")));
         };
+        if abs_path.starts_with("<node_internals>") {
```

Comment:
Could we instead ensure that `abs_path` is indeed absolute? `Path::is_absolute` should return false for <node_internals>, plus it lets us avoid lookups with any other non-abs paths.
https://play.rust-lang.org/?version=stable&mode=debug&edition=2024&gist=c1732d529ededc3ee71b1d2b164d8022

Reply:
Sorry, this auto-merged before I saw your comment! That makes sense, will follow up.